### DISCUSSION TITLE
Crashes after update KB5025305

### DISCUSSION THREAD

**Discussion by Willgefortz** [valinet/ExplorerPatcher#1389] [General]
ExplorerPatcher crashes after few clicks on taskbar.
Windows 11 22621.1635
It started after Windows update **KB5025305.**
Re-installation and reset to defaults didn't help.
Tried with ExplorerPatcher 22621.1555.55.2 and 22621.1555.55.1

**Comment by cheeseandjamsandwich** [comment]
Mine is crashing as soon as the desktop loads after signing in.
Imediate flashing, crashing, and then we get the new, and very much appreciated crashing aware pop-up,  Thanks Valinet!
I then just ctrl-shift-esc to start task manager and then 'run new task' 'explorer' and the old crappy taskbar returns, so the machine is still usable, just with the crap UX.
Classic shell's start button doesn't work if clicked on, but the keyboard's win key does, brings up classic shell's start menu.  Usable.
And obvs 7+TT stops working.

So it's annoying, but totally usable.
And it all comes with the territory of fixing M$'s crappy shell changes...  They will keep changing things forever.   Luckily we have solutions to return productivity.
